Dockerfile

```diff
@@ -1,10 +1,6 @@
-FROM golang:1.12.4-stretch
-RUN apt update 
-RUN apt install -y make
+FROM golang:1.12-buster
 RUN mkdir /app 
 WORKDIR /app 
 COPY . .
-RUN make test
-RUN make build
 EXPOSE 4455 5544
-CMD ["./bin/boxmetrics-agent", "start"]
\ No newline at end of file
+CMD ["./bin/boxmetrics-agent"]
```

Comment:
Why we no longer have to run "start" ? Where is it done now ?

Reply:
In order to build the pipeline, I update the application to do nothing for now. Refactor features will be add gradually